distribution/ddf-common/src/main/resources/security/default.policy

```diff
@@ -297,6 +297,10 @@ grant codeBase "file:/admin-core-insecuredefaults/org.apache.karaf.decanter.sche
     permission java.io.FilePermission "${ddf.home.perm}etc${/}users.properties", "write";
 }
 
+grant codeBase "file:/org.codice.thirdparty.commons-httpclient" {
```

Comment:
✏️ I just realized, after I approved (🤦‍♂️ ), that you can/should reuse another grant block in here. Every new grant block slightly affects the performance of the running system.
You should add the bundle `org.codice.thirdparty.commons-httpclient` to the end of the list of bundles in this block
```
grant codeBase "file:/org.apache.servicemix.bundles.commons-httpclient/org.apache.camel.camel-http/spatial-wfs-v1_0_0-source/spatial-wfs-v2_0_0-source/spatial-wfs-v1_1_0-source/org.apache.ws.xmlschema.core/security-rest-cxfwrapper" {
    permission java.io.FilePermission "${ddf.home.perm}etc${/}keystores${/}-", "read";
}
```